Localization/de-DE/home.yaml

```
@@ -0,0 +1,46 @@
+---
+url: "https://foldingathome.org/"
+language: "de-DE"
+text: [
+# START FOLDING NOW bold text
+#"START FOLDING NOW",
+"JETZT MITFALTEN",
+
+# Associated normal text
+#"Find the version of the software you prefer and get started. Downloading Folding@home is completely free, easy to install and safe to use.#
+#
+#Available for:
+#
+#Linux
+#
+#Windows
+#
+#Mac",
+"Suchen Sie sich die Version der Software aus, die ihnen am liebsten ist und legen Sie los. Folding@home herunterzuladen ist komplett kostenlos, es ist einfach zu installieren und die Nutzung ist sicher.
```

```suggestion
"Suchen Sie sich die Version der Software aus, welche Ihnen am liebsten ist und legen Sie los. Folding@home ist komplett kostenlos, einfach zu installieren und die Nutzung ist sicher.
```
- Changed _die_ zu _welche_
- Uppercase _Ihnen_ 
- Removed some words - imho easier to read and more /marketing/ potential